### ISSUE TITLE
How to manually add an app to pinned app list

### ISSUE THREAD

**Issue by greenlaw110** [jake-phy/WindowIconList#133]
I got a screen capture application, and when I launch it, it will just display the blocks to capture the screen. However it will not be displayed as an icon is the window list. I am wondering if there is a way for me to manually create a launcher for application and make it pinned app in the window list?

**Comment by jake-phy** [comment]
Yes if you look in the /usr/share/applications folder, you can open one of those launchers and change the values and then save to to the /home/{User}/.local/share/applications folder.  Once that is done you should be able to add it to the start menu with which you can then pin to the app list.  i will work on an easier way to do that once I get this version out.

**Comment by lukaszpolowczyk** [comment]
@jake-phy This method work in running app. But this method not hold window.
For example desktop file has exec=firefox -p lukasz
This launcher run firefox but in new icon, independent of launcher.

The same with Firefox Nightly. Despite Nightly is a separate program - new icon glue with "firefox -p lukasz".

This is worse than Windows 7. There was difficult. Here is a terribly difficult.
